builder.js

```diff
@@ -133,5 +134,11 @@ module.exports = {
         },
       }
     ]
-  }
+  },
+  resolve: {
+    fallback: {
+      "stream": require.resolve("stream-browserify"),
```

Comment:
what is this?

Reply:
Webpack 5 does not include polyfills by default but `stream` and `timers` are needed by dependencies of bio-parsers.
So I added them manually. 
error if not added e.g.:
```
WARNING in ./node_modules/sax/lib/sax.js 166:13-37
Module not found: Error: Can't resolve 'stream' in '/var/www/html/elabftw/node_modules/sax/lib'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "stream": require.resolve("stream-browserify") }'
        - install 'stream-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "stream": false }
 @ ./node_modules/xml2js/lib/parser.js 36:8-22
 @ ./node_modules/xml2js/lib/xml2js.js 27:11-30
 @ ./node_modules/bio-parsers/es/parsers/snapgeneToJson.js 358:0-28 421:4-22
 @ ./node_modules/bio-parsers/es/parsers/anyToJson.js 50:45-59 201:0-46
 @ ./src/ts/ove.js
 @ ./src/ts/edit.ts 2:0-45 43:24-44
```